src/main/java/net/coderbot/iris/uniforms/CapturedRenderingState.java

```diff
@@ -99,4 +102,19 @@ public ValueUpdateNotifier getBlockEntityIdNotifier() {
 	public int getCurrentRenderedEntity() {
 		return currentRenderedEntity;
 	}
+
+    public float getCurrentAlphaTest() {
+		return currentAlphaTest;
+    }
+
+	public void setCurrentAlphaTest(float alphaTest) {
+		this.currentAlphaTest = alphaTest;
+		if (this.alphaTestListener != null) {
+			this.alphaTestListener.run();
+		}
```

Comment:
This isn't needed since this can only ever be called before we set up the uniforms anyways. It's valid for a dynamic uniform to have an update notifier that's just `() -> {}`.